packages/queries/src/types/ColumnValue.ts

```diff
@@ -14,15 +14,11 @@ export type ColumnValue<
     DB,
     TTable extends Table<DB>,
     TColumn extends Column<DB, TTable>,
-> =
-    // Case 1: The value is a ColumnType
-    DB[TTable][TColumn] extends Selectable<infer T>
-        ? T
-        : // Case 2: The value is a nullable ColumnType
-          DB[TTable][TColumn] extends Selectable<infer T> | null
-          ? T | null
-          : DB[TTable][TColumn];
+> = DB[TTable] extends { columns: infer TColumnDef }
+    ? ColumnType<TColumnDef, TColumn>
+    : never;
 
-type Selectable<T> = {
-    readonly __select__: T;
-};
+type ColumnType<
```

Comment:
Please add some documentation

Reply:
You changed the documentation for ColumnValue (which I thought was already good enough) but you didn't add any do ColumnType which was my request 😅

Reply:
Ahh.. that explains it, I didn't realize you were referring to ColumnType. Will do so now!